README.md

```diff
@@ -60,6 +60,12 @@ go check.HandleTimeout(checkPluginTimeoutInSeconds)
 
 ## Thresholds
 
+Threshold objects represent Nagios-style thresholds that have methods to evaluate if a given input is within the range.
+
+They can be created with the ParseThreshold parser.
+
+https://nagios-plugins.org/doc/guidelines.html#THRESHOLDFORMAT
```

Comment:
Please link to this or the RFC draft: https://github.com/monitoring-plugins/monitoring-plugin-guidelines/blob/main/definitions/01.range_expressions.md

Reply:
Done